src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -5,7 +5,7 @@
 import mate.academy.model.Movie;
 
 public interface MovieDao {
-    Movie add(Movie movie);
+    Movie add(Movie entity);
```

Comment:
Why did you rename this? It's better to differentiate entities.

Reply:
I saw that in previsious task
I can cut and past the code without changing